charts/redis/templates/redis-dc.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.managed_redis.disabled }}
```

Comment:
This if statement is not required here. If we want a managed DB, we won't even deploy this entire chart. We can remove it.

Reply:
Done

Reply:
Done